src/lib/users/dialogs/inst_user_first.rb

```diff
@@ -283,9 +263,18 @@ def root_user
       users_config.users.find(&:root?)
     end
 
-    # TODO: let's simply return false for the time being
-    def imported?(_user)
-      false
+    def user_imported?
+      user&.source == :local
```

Comment:
I am not sure if this will stay true. As common installation also creates local users.

Reply:
Yes, this is a temporary solution and I wanted to discuss (after some manual testing) what was the best option. So far I fear we are ignoring the `#source` attribute and we should maybe even rethink it.